miniapps/zora-nft-minter/src/view.ts

```diff
@@ -0,0 +1,64 @@
+import { ModElement } from "@mod-protocol/core";
+
+const view: ModElement[] = [
+  {
+    type: "card",
+    imageSrc: "{{embed.metadata.nft.mediaUrl}}",
```

Comment:
this data may not be available for all zora urls - for example in the race condition that it's not been indexed yet. might need to put a check